### ISSUE TITLE
PyPI homepage links lead to malicious websites

### ISSUE THREAD

**Issue by rabyj** [lmacken/pyrasite#151]
PyPI : `https://pypi.org/project/pyrasite/`

Both homepage links lead to `http://pyrasite.com/` which leads to `http://ww1.pyrasite.com/` which leads to various malicious websites.

Please investigate asap

**Comment by gl-yziquel** [comment]
Yes. Indeed. Annoying.

**Comment by rabyj** [comment]
I don't think this should be closed @lmacken, the PyPI homepage still links lead to malicious websites. It's blocked on my domain.

Scam score: https://www.scamadviser.com/check-website/dypigu.com

How it was blocked on my end (on a university domain):
URL: dypigu.com/f.php?e=B4djN5cYpnHdgZRFKLxlPn49fjgySTRJckQ2N3BwSi9iNmsvaFgvTFFuL3VndmsyTkgrdWdUOEFtelBtUnUwejZaNTNQdi9PZlR1dmxsbUlnblgyUjlzSEtuWG5BU2ljMkw0LytSdGJmTXM2ejJXNU9nY3NiYkZvR1FrMTJWUWJWdzNQeU11d3FrS1JjWUtJYUpQNm1DNjVVVThRQVN3YWZxeUVyVTlodWRaUTZtYTlFL2QwMU0zbjVSSTY2dW5Jayt6Q091UTd2VGRscm4xRWlIOVBOSmR5WVpqQitwcWtxRlBrWHhzS2FvbXRyVkJLRDJ5YUU5VFkrZGgzT2Vmbnh1YzE4YWxpT2Fvek9SY0d2bHlIcU5kYVVHaks3U1N4dHl4WElWNnpQYlBkcExwKzdLUkRGTmJsVDFCaWhhZnpDaHQ5TFFVN1A4N040UGZFSFJOeFJ6dStaTlg2WFRGUk56MTRwWERONExGRUdhRXR2RjlJY21EZDAxZ3ROMU9zUzZGR3crK2N5NmMxYTVRSm15dWFhcGNIUG5UMVpsWmNSU2Znb0hFU1BMZWFNWGJCc0I5QUVCRUNmMTNMU2kyVXluZG1QMCtpcVJqbHlGeTdBa285cS9BN3pFeXJvbStuc1JwM1RZcTRRPT0%3D
Categorie: Malicious Websites

**Comment by dwpaley** [comment]
Same issue for me and the bad url is also listed on this repo: 

<img width="363" alt="Image" src="https://github.com/user-attachments/assets/51bec00d-c050-4d90-be04-b52c118efa99" />
